relnotes/pkgs-variable.feature.md

```diff
@@ -1,11 +1,15 @@
-* `PKG_FILES`
-
-  This variable holds all package definitions to be build. The purpose is to allow one to exclude
-  some packages from being built if need be, for example for `F=production` or `DVER=2`
-  only packages.
-  It can be filtered-out, e.g:
-  ```
-  ifeq ($(DVER),1)
-  PKG_FILES := $(filter-out $(PKG)/D2OnlyApp.pkg,$(PKG_FILES))
-  endif
-  ```
+## Packages to be build can be specified via `PKG_FILES`
+
+`mkpkg`
+
+A new `PKG_FILES` variable holds all package definitions to be build.
+
+The purpose is to allow one to exclude some packages from being built if need be, for example for `F=production` or `DVER=2` only packages.
+
+The `PKG_FILES` variable comes populated with `pkg/*.pkg` by default, but it can
+be also filtered-out, e.g:
```

Comment:
The "also" makes little sense to me. Maybe you mean "[...] by default, it can be overriden or filtered out as follows:", or "[...] by default, and can be filtered out as follows:"

Reply:
No, I mean it like that. I think I already had a discussion with a similar case. I guess this is probably a valid use if translated literally to Spanish, but something that doesn't work out in English :P

I will use something similar to what you suggest.